### PR TITLE
feat: add version number in settings

### LINKED ISSUE #30: Add a version number on the extension
It will avoid strange workarounds like this one : https://github.com/open-wc/locator/issues/29#issuecomment-696218462

### PR DESCRIPTION
**Pull request by jorgecasar**
And also I change a bit settings styles.

generic-switch was fixed to 16px and the second toggle are bigger than that. I will open a PR to generic-switch 👍 

image

Close #30

### CODE REVIEW

**Comment by thepassle** [comment]
Could you post a screenshot of what it looks like?

**Comment by jlengrand** [comment]
@jorgecasar going hard after those sweet Hacktoberfest PRs XD. Keep it going mate!

**Comment by jorgecasar** [comment]
Added PR description

File: src/custom-elements-locator.js [hunk 1]
```diff
@@ -73,6 +76,10 @@ class CustomElementsLocator extends LitElement {
 
     this.__gotLatestElements = this.createDeferredPromise();
 
+    fetch('../manifest.json')
```
Comment (jlengrand) [inline src/custom-elements-locator.js:79]: Awesome, I didn't even know that was possible 👍

**Review by jlengrand — APPROVED** [review]

**Comment by jlengrand** [comment]
@thepassle LGTM. I added the hacktoberfest label to the issue and the PR as well

**Pull request merged**